client/my-sites/site-settings/delete-site-options/index.jsx

```diff
@@ -90,11 +90,15 @@ module.exports = React.createClass( {
 					<div className="delete-site-options__content">
 						<h2 className="delete-site-options__section-title">{ strings.deleteSite }</h2>
 						<p className="delete-site-options__section-desc">
-							{ this.translate( 'All your posts, images, data, and this site\'s address ({{siteAddress /}}) will be gone forever.', {
-								components: {
-									siteAddress: <strong>{ selectedSite.slug }</strong>
+							{ this.translate(
+								'All your posts, images, and data will be deleted. ' +
+								'And this site’s address ({{siteAddress /}}) will become inactive.',
+								{
+									components: {
+										siteAddress: <strong>{ selectedSite.slug }</strong>
```

Comment:
There's been feedback that the wording should maybe be "will be lost" rather than "will become inactive" and I like this proposed edit.

This will have been incorrect previously too, but is there something we can do to force the *.wordpress.com domain in the message? I tested with a site that had a domain, and it told me the domain would be inactive; if we update wording to "will be lost," that's pretty scary and not actually accurate. Ideally I suppose we'd check for the existence of a domain and nudge the user to remap the domain to another site or change DNS before deleting, and otherwise display the *.wordpress.com domain here. Doing the latter may be a nice interim improvement.